
message PlayerLoginResquest
{
    required string token = 1;
}

message CreatePlayerRequest{
     optional string nickname = 1;//角色昵称
}

